config/config.json

```diff
@@ -182,6 +182,13 @@
       "tag": "package-2015-06-01-preview"
     }
   },
+  "azure_mgmt_automanage": {
+    "azure_mgmt_automanage_2020_06_30_preview": {
+      "markdown": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/38907c94084f9f6222206872b99674d3d48cd3e3/specification/automanage/resource-manager/readme.md",
```

Comment:
Usually we will only use one latest commit version. But it's fine unless this is the required commit version.